src/numeric-menu/__tests__/numeric-menu.spec.ts

```diff
@@ -27,11 +27,13 @@ describe('NumericRefinementList', () => {
     expect(fixture).toMatchSnapshot();
   });
 
-  it('should refine() when item click', () => {
+  it('should refine() when label click', () => {
```

Comment:
The test does not really ensure that we correctly setup the listener. We ensure that a click on `label` triggers `refine` but without the previous implementation it's already the case.

Reply:
I added an additional test to check that we bug is correctly fixed when you click on the label a second time.